app/views/questions/_comment.html.erb

```diff
@@ -23,10 +23,13 @@
         <%= render partial: "questions/form", locals: { comment: comment } %>
       </form>
     </div>
-    
+
   </div>
   <div class="inline pull-xs-right" style="vertical-align: top">
-    <p><% if current_user && current_user.uid == comment.uid %>
+    <p><% if answer_id == 0 && current_user && current_user.uid == comment.uid %>
```

Comment:
Hi, @Gauravano -- could we expand this to include admins and moderators?

Reply:
Hmm..actually I was thinking whether it would be appropriate to move any user's comment to answer section without any notice.We can add a suggest user feature and then it would be user's call.What do you think? But, if you want I can implement it like it's there for delete